packages/solarwinds-apm/src/shared/config.ts

```diff
@@ -174,7 +163,52 @@ export const schema = (defaults: Defaults) =>
       ),
       exportLogsEnabled: v.optional(schemas.boolean, false),
 
-      transactionName: v.optional(schemas.stringy),
+      transactionName: v.optional(
+        v.union([
+          v.pipe(
+            v.string(),
+            v.transform((name) => () => name),
```

Comment:
To confirm my understanding, an env var setting e.g. `SW_APM_TRANSACTION_NAME='[{"naive":1},{"array":2}]'` would always be treated as string and processed here?

Reply:
Yeah, unless we were to start parsing every environment variable as JSON which seems like it might cause a lot of unintended consequences